app/views/home/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
why?

Reply:
I have removed home.css so added this stylesheet for styling purposes.Currently working on styling.

Reply:
why importing bootstrap in application.js and application.scss then?

Reply:
understood now.